src/Microsoft.Restier.EntityFramework/Model/ModelProducer.cs

```diff
@@ -124,17 +121,19 @@ private ModelProducer()
         }
 
         private static IEdmEntityType CreateEntityType(
-            MetadataWorkspace efModel, EntityType efEntityType,
-            EdmModel model, out List<EdmStructuralProperty> concurrencyProperties)
+            MetadataWorkspace efModel,
+            EntityType efEntityType,
+            EdmModel model,
+            Dictionary<MetadataItem, IEdmElement> elementMap,
```

Comment:
Since these are private methods, maybe better to use concrete type for maximized performance?